# Force Sensor
topic_force_sensor: "/ethdaq_data"

# Position and Velocity Control
topic_joint_trajectory: "/scaled_pos_joint_traj_controller/command"
topic_action_trajectory: "/scaled_pos_joint_traj_controller/follow_joint_trajectory"
topic_joint_group_vel_controller: "/joint_group_vel_controller/command"

# Joint State Feedback
topic_joint_states: "/joint_states"
